sql/201709091029_InsertProduct.sql

```diff
@@ -0,0 +1,2 @@
+Insert into Product 
+values ('Coffee Cups','25 count 16oz, Paper with Lids, Biodegrable','4.23')
```

Comment:
Is this description to long for the field length?

Reply:
Yes, we replaced this script with another scipt

Reply:
OK, if this file is outdated then go ahead and delete or modify it.